src/services/logging/sentry/log.ts

```diff
@@ -4,7 +4,7 @@ export const error = (
   error: Error,
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   setup: (scope: Sentry.Scope) => void = () => {},
-  severity: Sentry.Severity.Critical | Sentry.Severity.Fatal | Sentry.Severity.Error = Sentry.Severity.Error
+  severity: 'fatal' | 'error' = 'error'
```

Comment:
Looking at the link you have added, it will be cleaner to use, e.g. for the warning:

```
export const warningLevel: SeverityLevel = 'warning';
```

And then:

```
export const warn = (warning: string) => {
  Sentry.withScope(scope => {
    scope.setLevel(warningLevel);
    Sentry.captureEvent({ message: warning });
  });
};
```

Naming and placing might need to be changed, but the idea is to avoid using string literals directly and to wrap them up in constants (and that's actually what they suggested:

Severity, SeverityLevel, and SeverityLevels
We deprecated the Severity enum in @sentry/types and it will be removed in the next major release. We recommend using string literals (typed as SeverityLevel) to save on bundle size.

// New in 6.17.5:
import { SeverityLevel } from '@sentry/types';

const levelA = "error" as SeverityLevel;

const levelB: SeverityLevel = "error"

// Before:
import { Severity, SeverityLevel } from '@sentry/types';

const levelA = Severity.error;

const levelB: SeverityLevel = "error"

Reply:
The benefit is that you are actually still using Sentry types. If they change something in the future (possible, albeit unlikely), e.g. drop info level, you will immediately see that upon SDK update when you build the solution. With direct string literals, untyped, you will end up seeing this error in production.

Reply:
Good, understood,
I'll push something in a few minutes